### ISSUE TITLE
Microprofile Rest Client (CDI): mpConfig property "proxyAddress" not respected

### ISSUE THREAD

**Issue by pugmas** [OpenLiberty/open-liberty#26722]
**Describe the bug**  
The mpConfig property 
`MyRestClient/mp-rest/proxyAddress=http://my.proxy.net:7808`
has no effect. [Spec](https://github.com/eclipse/microprofile-rest-client/blob/b6eb2f651c0cbb19f0300eba5c6a3242e9e46019/spec/src/main/asciidoc/cdi.asciidoc#support-for-microprofile-config)

Other properties are recognized. E.g.: 
`MyRestClient/mp-rest/uri=https://my.api.com`

**Expected behavior**  
The Microprofile Rest Client should use the configured proxy-server.

**Diagnostic information:**  
 - OpenLiberty Version: 23.0.0.9  with webProfile-10.0 and microProfile-6.0
 - Java Version: 17

**Additional context**  
The following does work:
```
var client = RestClientBuilder.newBuilder()
                    .baseUri(URI.create("https://my.api.com"))
                    .proxyAddress("http://my.proxy.net:7808")
                    .build(MyRestClient.class);
```

**Comment by WhiteCat22** [comment]
This should be fixed in 23.0.0.12

**Comment by pugmas** [comment]
Thank you! That was pretty fast, great!
